pcdsdevices/lodcm.py

```diff
@@ -60,7 +97,44 @@ def __init__(self, prefix, *args, **kwargs):
         super().__init__(prefix, *args, **kwargs)
 
 
-class LODCM(BaseInterface, Device):
+class CHI1(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class CHI2(InOutRecordPositioner):
+    states_list = ['C', 'Si']
+    in_states = ['C', 'Si']
+    out_states = []
+
+
+class H2N(InOutRecordPositioner):
+    states_list = ['C', 'Si']
```

Comment:
There's probably a less wordy way to set all of these up, because of https://github.com/pcdshub/pcdsdevices/blob/757834326d6b95a1ff49507999502b2f81e8f3b3/pcdsdevices/state.py#L36-L37
This is related to #543
Might not be in scope for this PR